exporters/otlp/otlptrace/README.md

```diff
@@ -0,0 +1,35 @@
+# OpenTelemetry-Go OTLP Span Exporter
+
+[![Go Reference](https://pkg.go.dev/badge/go.opentelemetry.io/otel/exporters/otlp/otlptrace.svg)](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlptrace)
+
+OpenTelemetry Protocol (OTLP) Span Exporter.
+
+## Installation
+
+```
+go get -u go.opentelemetry.io/otel/exporters/otlp/otlptrace
+```
+
+## Example
+
+To constructs a new OTLP trace Exporter, you can follow this
```

Comment:
should be "span exporter"

Reply:
You mean it should be change as

Reply:
I am lazy and I am suggesting https://github.com/open-telemetry/opentelemetry-go/pull/2053/files#r679920638 😉